cmd/pulumi-test-language/providers/simple_provider.go

```diff
@@ -63,6 +63,9 @@ func (p *SimpleProvider) GetSchema(
 	pkg := schema.PackageSpec{
 		Name:    "simple",
 		Version: "2.0.0",
+		Meta: &schema.MetadataSpec{
```

Comment:
Remove this. Conformance testing adds it to all the schemas anyway it doesn't need setting here.

Reply:
program-gen always got `schema.SupportPack == false` so I though it was because a schema was being serialized and deserialized

Reply:
Ah we can fix that in the loader see providerLoader.LoadPackageReference

Reply:
Fixed ✅